### PR TITLE
feat: Quantity Selector

### PR DESCRIPTION
**Pull request by kibo-sushant**
Written StoryBookTestCases
Written Unit TestCases
Tested as per user acceptance test criteria

### CODE REVIEW

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]
Please check

File: components/common/QuantitySelecotor/QuantitySelector.test.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+// import StoryBook stories
+const { Default } = composeStories(stories)
+
+describe('[components] - QuantitySelector', () => {
+  test('should render component', () => {
+    // arrange
+    const onIncreaseMock = jest.fn()
+    const onDecreaseMock = jest.fn()
+    render(
+      <QuantitySelecotor quantity={1} onIncrease={onIncreaseMock} onDecrease={onDecreaseMock} />
```
[the remark this answers is not in the document]
Comment (kibo-sushant) [reply components/common/QuantitySelecotor/QuantitySelector.test.tsx:17]: As I have to mock eventHandlers, reusing StoryBook stories will not serve the purpose

File: components/common/QuantitySelecotor/QuantitySelecotor.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        textAlign: 'center',
+      },
+    }}
+    sx={{ width: '46px', height: '37px' }}
+  />
+)
+
+// Component
+const QuantitySelecotor = (props: QuantitySelecotorProps) => {
+  const { quantity = 1, onIncrease, onDecrease } = props
```
[the remark this answers is not in the document]
Comment (kibo-sushant) [reply components/common/QuantitySelecotor/QuantitySelecotor.tsx:49]: As I don't want to maintain the state inside the component, used this approch

**Review by kibo-chandradeeptalaha — APPROVED** [review]
Please remove the unused import and that's good to go.

**Review by kibo-kevinwatts — APPROVED** [review]
Very clean code! :)

There is a typo, QuantitySelecotor should be QuantitySelector

**Pull request merged**